assets/airports/rjaa.json

```diff
@@ -0,0 +1,1153 @@
+                "34L": [["ARIES", "A40+"], "A4L11", "A4L12", ["A4L13", "A70"], "A4L14", "JELLY"],
+                "34R": ["BOXER", "A4R21", "A4R22", ["A4R23", "A70"], "SOHSA"]
+            },
+            "exitPoints": {
+                "CUPID": ["CUPID"]
+            },
+            "draw": [
+                ["A6L21", "CUPID"],
+                ["A6R12", "CUPID"],
+                ["JELLY", "CUPID"],
+                ["SOHSA", "CUPID"]
+            ]
+        },
+        "DAITO2": {
+            "icao": "DAITO2",
+            "name": "Daito Two",
+            "rwy": {
+                "16L": ["BEAMS", "A6L22", "DAITO"],
+                "16R": ["ASPEN", "A6R12", "DAITO"],
+                "34L": ["ASTRA", "AA433", "DAITO"],
+                "34R": ["BOXER", "AA433", "DAITO"]
+            },
+            "exitPoints": {
+                "MERCY": ["MERCY"]
+            },
+            "draw": [
+                ["A6L22", "DAITO", "MERCY"],
+                ["A6R12", "DAITO"],
+                ["ASTRA", "AA433", "DAITO"],
+                ["BOXER", "AA433"],
+                ["MERCY"]
+            ]
+        },
+        "SAKURA4": {
+            "icao": "SAKURA4",
+            "name": "Sakura Four",
+            "rwy": {
+                "16L": ["NRE"],
+                "16R": ["NRE"],
+                "34L": ["HKE", "NRE"],
+                "34R": ["HKE", "NRE"]
+            },
+            "exitPoints": {
+                "GOTEN": [["HME", "A200+"], ["GOTEN", "A240+"]],
+                "SZE": [["HME", "A200+"], "SZE"]
+            },
+            "draw": [
+                ["NRE", "TETRA"],
+                ["HKE", "NRE", "TETRA"],
+                ["TETRA", "HME"],
+                ["HME", "GOTEN*"],
+                ["HME", "SZE*"]
+            ]
+        },
+        "AKAGI2": {
+            "icao": "AKAGI2",
+            "name": "Akagi Two",
+            "rwy": {
+                "16L": [["ARWEN", "A70"]],
+                "16R": [["ARWEN", "A70"]],
+                "34L": [["ARIES", "A40+"], "TOADS", ["ARWEN", "A70"]],
+                "34R": [["CREEK", "A40+"], "METIS", ["ARWEN", "A70"]]
+            },
+            "exitPoints": {
+                "AKAGI": ["AKAGI"]
+            },
+            "draw": [
+                ["ARWEN", "AKAGI"],
+                ["ARIES", "TOADS", "ARWEN"],
+                ["CREEK", "METIS", "ARWEN"],
+                ["AKAGI"]
+            ]
+        },
+        "ROUSY2": {
+            "icao": "ROUSY2",
+            "name": "Rousy Two",
+            "rwy": {
+                "16L": [["ROUSY", "A70+"]],
+                "16R": [["ROUSY", "A70+"]],
+                "34L": [["NRE", "A28+"], ["ROUSY", "A70+"]],
+                "34R": [["HKE", "A28+"], ["ROUSY", "A70+"]]
+            },
+            "exitPoints": {
+                "SUNNS": [["SUNNS", "A190+"]]
+            },
+            "draw": [
+                ["NRE", "HKE", "ROUSY", "SUNNS*"]
+            ]
+        }
+    },
+
+    "stars": {
```

Comment:
I'm only seeing `ELGAR`, `TYLER`, `NORMA`, and `GEMIN` arrivals, and nothing on the AIS for `ABBOT`, `SWAMP`, and `LAKES`?